lib/synapse-plugins/system-managment.vala

```diff
@@ -455,7 +473,7 @@ namespace Synapse {
                     continue;
                 }
                 foreach (var matcher in matchers) {
-                    if (matcher.key.match (action.title)) {
+                    if (match(action, matcher)) {
```

Comment:
Missing space before bracket

Reply:
Good one, I've -re-checked the whole file for those.